scene/main/node.cpp

```diff
@@ -1364,6 +1364,34 @@ bool Node::has_node(const NodePath &p_path) const {
 	return get_node_or_null(p_path) != nullptr;
 }
 
+Node *Node::get_unique_node(const StringName &p_name) const {
+	if (!p_name) {
+		ERR_FAIL_V_MSG(nullptr, "'name' parameter cannot be empty");
+	}
+
+	const StringName key = StringName(UNIQUE_NODE_PREFIX + p_name);
```

Comment:
The problem with this approach is that this bit is actually quite inefficient compared to just requesting "%Node", so if we added this function and some users prefer it, it would be slower.

I am not sure how it can be done better, as it is not intended to be obtained like this.

Reply:
This is true, yeah. Please see the benchmark in https://github.com/godotengine/godot/pull/64274